packages/backend/src/controllers/orders.controller.ts

```
@@ -1,100 +1,99 @@
+ * @route GET /api/orders/:id
+ * @access Admin only
  * @param req Express Request
  * @param res Express Response
- * @returns 200 + array d'orders
+ * @returns 200 + Order | 404 | 500
  */
-export const getAllOrders = async (_req: Request, res: Response): Promise<void> => {
-  const orders = await ordersService.getAllOrders();
-  res.status(200).json(
-    orders.map(order => ({
+export const getOrderById = async (req: Request, res: Response): Promise<Response | void> => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)